components/admin_console/billing/company_info_edit.tsx

```diff
@@ -54,12 +57,14 @@ const CompanyInfoEdit: React.FC<Props> = () => {
         return (event: React.ChangeEvent<HTMLInputElement>) => {
             setStateFunc(event.target.value);
             setValidation();
+            setContentChanged(true);
         };
     };
 
     const updateNumEmployees = (event: React.ChangeEvent<HTMLInputElement>) => {
         if (event.target.value) {
             setNumEmployees(parseInt(event.target.value, 10));
+            setContentChanged(true);
```

Comment:
from a purely logic perspective, I would expect `setContentChanged` to be true even if the input is empty/wrong/malformed.

Reply:
Definitely! Nice catch :)